src/java/picard/analysis/MeanQualityByCycle.java

```diff
@@ -52,14 +52,25 @@
  * @author Tim Fennell
  */
 @CommandLineProgramProperties(
-        usage = "Program to generate a data table and pdf chart of " +
-                "mean base quality by cycle from a SAM or BAM file.  Works best on a single lane/run of data, but can be applied to" +
-                "merged BAMs. Uses R to generate chart output.",
-        usageShort = "Writes mean quality by cycle for a SAM or BAM file",
+        usage = MeanQualityByCycle.USAGE_SUMMARY + MeanQualityByCycle.USAGE_DETAILS,
+        usageShort = MeanQualityByCycle.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
 public class MeanQualityByCycle extends SinglePassSamProgram {
-
+    static final String USAGE_SUMMARY = "Writes mean quality by cycle for a SAM or BAM file";
+    static final String USAGE_DETAILS = "Tool generates a data table and chart of mean quality by cycle from a BAM file." +
+            "  Works best on a single lane/run of data, but can be applied to merged BAMs.  <br /><br />" +
+            "This metric gives an overall snapshot of sequencing machine performance, as base quality scores" +
+            " tend to decrease towards the end of a run. <br /><br />" +
+            "" +
```

Comment:
Two separate sentences?  
"This metric gives an overall snapshot of sequencing machine performance.  Base quality scores" +
+            " tend to decrease towards the end of a run. <br /><br />" +